HT4/HT4/Program.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+
+namespace HT4
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            Console.WriteLine("Укажите размер массивов");
+            int num = Convert.ToInt32(Console.ReadLine());
+            char[] alphabet = { 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z' };
+            char count = ' ';
+            int result = 0;
+            char[] input = { 'a', 'e', 'i', 'd', 'h', 'j' };
+            int c_even = 0;
+            int c_odd = 0;
+            if (num > 0)
+            {
+                string[] arr_even = new string[num];
+                string[] arr_odd = new string[num];
+                Console.WriteLine(" ");
+                Console.WriteLine("четные числа");
+                for (int i = 0; i < arr_even.Length; i++)
+                {
+                    int a = new Random().Next(1, 26);
+                    if (a % 2 == 0)
+                    {
+                        arr_even[i] = a.ToString();
+                        Console.Write(arr_even[i] + " ");
+                    }
+                    else
+                    {
+                        i--;
+                    }
+                }
+
+                Console.WriteLine(" ");
+                Console.WriteLine("нечетные числа");
+
+                for (int i = 0; i < arr_odd.Length; i++)
+                {
+                    int b = new Random().Next(1, 26);
+                    if (b % 2 == 1)
+                    {
+                        arr_odd[i] = b.ToString();
+                        Console.Write(arr_odd[i] + " ");
+                    }
+                    else
+                    {
+                        i--;
+                    }
+                }
+
+                Console.WriteLine(" ");
+                Console.WriteLine("Новый массив из букв для четных чисел:");
+                for (int i = 0; i < arr_even.Length; i++)
```

Comment:
The code can be separated into the standalone methods (functions), so it would be much easier to read and understand